src/libraries/Common/src/System/Buffers/ArrayBufferWriter.cs

```diff
@@ -186,5 +196,10 @@ private static void ThrowInvalidOperationException_AdvancedTooFar(int capacity)
         {
             throw new InvalidOperationException(SR.Format(SR.BufferWriterAdvancedTooFar, capacity));
         }
+
+        private static void ThrowOutOfMemoryException(uint capacity)
+        {
+            throw new OutOfMemoryException(SR.Format(SR.BufferMaximumSizeExceeded, capacity));
```

Comment:
Where is `BufferMaximumSizeExceeded` defined?

Reply:
I just commited the resx file